roles/ideascube/tasks/ideascube.yml

```diff
@@ -34,7 +34,7 @@
 #   tags: ['master', 'custom','update']
 
 - name: Install ideascube
-  apt: name=ideascube force=yes update_cache=yes dpkg_options="force-confnew" state=latest
```

Comment:
This could be indeed avoid. apt-get update is done in system role. We just have to make sure it works !